src/test/java/com/wikia/webdriver/testcases/adstests/TestUrlBuilder.java

```diff
@@ -259,121 +263,109 @@
-                            .getUrlForPath((String) data[0], (String) data[10], (String) data[1]),
-                    (String) data[4]);
-            Assertion.assertEquals(new UrlBuilder("preview", (Boolean) data[2], (Boolean) data[3])
-                            .getUrlForPath((String) data[0], (String) data[10], (String) data[1]),
+            Assert.assertEquals(c.newInstance((String) data[0], "prod", (Boolean) data[2], (Boolean) data[3], (String) data[10])
+                            .getUrlForPath((String) data[1]), (String) data[4]);
+            Assert.assertEquals(c.newInstance((String) data[0], "preview", (Boolean) data[2], (Boolean) data[3], (String) data[10])
+                            .getUrlForPath((String) data[1]),
                     (String) data[5]);
-            Assertion.assertEquals(new UrlBuilder("sandbox", (Boolean) data[2], (Boolean) data[3])
-                            .getUrlForPath((String) data[0], (String) data[10], (String) data[1]),
+            Assert.assertEquals(c.newInstance((String) data[0], "sandbox", (Boolean) data[2], (Boolean) data[3], (String) data[10])
+                            .getUrlForPath((String) data[1]),
                     (String) data[6]);
-            Assertion.assertEquals(new UrlBuilder("sandbox-mercurydev", (Boolean) data[2], (Boolean) data[3])
-                            .getUrlForPath((String) data[0], (String) data[10], (String) data[1]),
+            Assert.assertEquals(c.newInstance((String) data[0], "sandbox-mercurydev", (Boolean) data[2], (Boolean) data[3], (String) data[10])
+                            .getUrlForPath((String) data[1]),
                     (String) data[7]);
-            Assertion.assertEquals(new UrlBuilder("dev-dmytror", (Boolean) data[2], (Boolean) data[3])
-                            .getUrlForPath((String) data[0], (String) data[10], (String) data[1]),
+            Assert.assertEquals(c.newInstance((String) data[0], "dev-dmytror", (Boolean) data[2], (Boolean) data[3], (String) data[10])
+                            .getUrlForPath((String) data[1]),
                     (String) data[8]);
-            Assertion.assertEquals(new UrlBuilder("staging", (Boolean) data[2], (Boolean) data[3])
-                            .getUrlForPath((String) data[0], (String) data[10], (String) data[1]),
+            Assert.assertEquals(c.newInstance((String) data[0], "staging", (Boolean) data[2], (Boolean) data[3], (String) data[10])
+                            .getUrlForPath((String) data[1]),
                     (String) data[9]);
         }
     }
 
     @Test(groups = {"TestUrlBuilder", "unitTests"})
-    public void getWikiaUrlTests() {
+    public void getWikiaUrlTests() throws Exception{
+        Constructor<UrlBuilder> c = getUrlBuilderConstructor();
 
-        Assertion.assertEquals(new UrlBuilder("preview", false).getWikiGlobalURL(),
+        Assert.assertEquals(c.newInstance("mediawiki119","preview", false, false, DEFAULT_LANGUAGE).getWikiGlobalURL(),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "mediawiki119" 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)